src/Term.php

```diff
@@ -80,10 +80,13 @@ protected function __construct()
 
     /**
      * @internal
-     * @param \WP_Term the vanilla WP term object to build from
+     *
+     * @param \WP_Term $wp_term The vanilla WordPress term object to build from.
+     * @param array    $options Optional. Unused. An array of options.
+     *
      * @return \Timber\Term
      */
-    public static function build(WP_Term $wp_term, array $_options = []): self
+    public static function build(WP_Term $wp_term, array $options = []): self
```

Comment:
Can you explain the $options parameter if unused?

Reply:
I think it's related to `Menu` where it can take an additional `$args` argument. But I agree this feels weird.

Reply:
Hm... but `Menu` is a separate class that uses a `WP_Term` under the hood, but not actually `Timber\Term`.

I guess it comes from a state where `Timber\Menu` extended `Timber\Term`, see https://github.com/timber/timber/commit/21445a1f5b50606fb3f8789c981f698a0437b2b9.

I’d say we can remove this unused parameter in a separate pull request. I’ll create one.

Good catch, @Levdbas!

Reply:
See https://github.com/timber/timber/pull/2754